src/collision/cm_model.c

```diff
@@ -422,6 +423,15 @@ static void Cm_LoadBspAreaPortals(const d_bsp_lump_t *l) {
 cm_bsp_model_t *Cm_LoadBspModel(const char *name, int64_t *size) {
 	void *buf;
 
+	// unload materials
+	for (int32_t i = 0; i < cm_bsp.num_surfaces; ++i) {
+		cm_bsp_surface_t *surf = &cm_bsp.surfaces[i];
+
+		if (surf->material) {
+			Cm_UnrefMaterial(surf->material);
+		}
+	}
+
```

Comment:
So in theory, the collision model could unload the map, but the materials could stay resident if quemap or the renderer or something still needed them? Seems robust.. hopefully doesn't provide rope to hang ourselves with :)